<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DCalibrationManageDetail</class>
 <widget class="QWidget" name="DCalibrationManageDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">image: none;
background: none;

background-color: rgba(0, 0, 0, 120);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>84</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>117</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>780</width>
       <height>600</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ffffff;
border-radius: 20px;
image: none;
border-image: none;
background-image: none;</string>
     </property>
     <widget class="QLabel" name="label_name">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>241</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;

width: 46px;
height: 23px;
font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #ffffff;
line-height: 34px;</string>
      </property>
      <property name="text">
       <string>校准品</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_no">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>480</y>
        <width>200</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(244, 96, 117, 255), stop:1 rgba(201, 67, 86, 255));
border-radius: 10px;
font-weight: bold;
color: white;
font-size: 32px;
font-family: PingFang SC;</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_yes">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>480</y>
        <width>200</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #22A4B3;
border-radius: 10px;

font-family: PingFang SC;
font-weight: bold;
font-size: 32px;
color: #FFFFFF;
line-height: 42px;</string>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>734</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_paramsName_1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 22px;
font-family: &quot;PingFang SC&quot;;
color: #666666;
</string>
           </property>
           <property name="text">
            <string>项目名称</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_params_1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: #FAFBFC;
border: 1px solid #DCDEE0;
border-radius: 6px;
font-size: 18px;
font-family: &quot;PingFang SC&quot;;
padding-left: 10px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_paramsName_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 22px;
font-family: &quot;PingFang SC&quot;;
color: #666666;
</string>
           </property>
           <property name="text">
            <string>校准品批号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_params_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: #FAFBFC;
border: 1px solid #DCDEE0;
border-radius: 6px;
font-size: 18px;
font-family: &quot;PingFang SC&quot;;
padding-left: 10px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_paramsName_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 22px;
font-family: &quot;PingFang SC&quot;;
color: #666666;
</string>
           </property>
           <property name="text">
            <string>样本管类型</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_params_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: #FAFBFC;
border: 1px solid #DCDEE0;
border-radius: 6px;
font-size: 18px;
font-family: &quot;PingFang SC&quot;;
padding-left: 10px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="styleSheet">
          <string notr="true">background: #FAFBFC;
border: 1px solid #DCDEE0;
border-radius: 6px;
font-size: 18px;
font-family: &quot;PingFang SC&quot;;
padding-left: 10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_paramsName_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 22px;
font-family: &quot;PingFang SC&quot;;
color: #666666;
</string>
           </property>
           <property name="text">
            <string>C0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="edit_c0">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: #FAFBFC;
border: 1px solid #DCDEE0;
border-radius: 6px;
font-size: 18px;
font-family: &quot;PingFang SC&quot;;
padding-left: 10px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_paramsName_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 22px;
font-family: &quot;PingFang SC&quot;;
color: #666666;
</string>
           </property>
           <property name="text">
            <string>C1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="edit_c1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: #FAFBFC;
border: 1px solid #DCDEE0;
border-radius: 6px;
font-size: 18px;
font-family: &quot;PingFang SC&quot;;
padding-left: 10px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_paramsName_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 22px;
font-family: &quot;PingFang SC&quot;;
color: #666666;
</string>
           </property>
           <property name="text">
            <string>C2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLineEdit" name="edit_c2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: #FAFBFC;
border: 1px solid #DCDEE0;
border-radius: 6px;
font-size: 18px;
font-family: &quot;PingFang SC&quot;;
padding-left: 10px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_paramsName_9">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 22px;
font-family: &quot;PingFang SC&quot;;
color: #666666;
</string>
           </property>
           <property name="text">
            <string>C3</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="edit_c3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: #FAFBFC;
border: 1px solid #DCDEE0;
border-radius: 6px;
font-size: 18px;
font-family: &quot;PingFang SC&quot;;
padding-left: 10px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_paramsName_11">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 22px;
font-family: &quot;PingFang SC&quot;;
color: #666666;
</string>
           </property>
           <property name="text">
            <string>C4</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="edit_c4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: #FAFBFC;
border: 1px solid #DCDEE0;
border-radius: 6px;
font-size: 18px;
font-family: &quot;PingFang SC&quot;;
padding-left: 10px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_paramsName_10">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 22px;
font-family: &quot;PingFang SC&quot;;
color: #666666;
</string>
           </property>
           <property name="text">
            <string>C5</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLineEdit" name="edit_c5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: #FAFBFC;
border: 1px solid #DCDEE0;
border-radius: 6px;
font-size: 18px;
font-family: &quot;PingFang SC&quot;;
padding-left: 10px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>60</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
background: none;
image: none;

background: #22a4b3;

border-top-left-radius: 20px;
border-top-right-radius: 20px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;</string>
      </property>
     </widget>
     <zorder>widget_5</zorder>
     <zorder>label_name</zorder>
     <zorder>btn_no</zorder>
     <zorder>btn_yes</zorder>
     <zorder>layoutWidget</zorder>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>117</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>84</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
